### ISSUE TITLE
Make a new "Unacceptable Behaviors" Document

### ISSUE THREAD

**Issue by kelle** [astropy/astropy-project#10]
So it turns out our Code of Conduct includes some aspirational elements which are more like "desirable" behaviors rather than "unacceptable behaviors." We might want to consider having a 2nd document which outlines the most egregious Code of Conduct violations. I propose this document could be called "unacceptable behaviors."

**Comment by kelle** [comment]
This is motivated by best practices discussed in Chapter 4 of this manual on Codes of Conduct: https://files.frameshiftconsulting.com/books/cocguide.pdf

**Comment by astrofrog** [comment]
I agree with this!

**Comment by kelle** [comment]
Also, need to think about reporting and aligning with Numfocus best practices.

**Comment by kelle** [comment]
@bsipocz has seen an example of an Unacceptable Practices document but can't find it.

**Comment by bsipocz** [comment]
It's the PSF one: https://www.python.org/psf/conduct/

**Comment by bsipocz** [comment]
(a conference also had a different, good one, but I cannot find that now)

**Comment by kelle** [comment]
@cadair has started convo with NumFOCUS to help out. Stuart, could you summarize?

**Comment by pllim** [comment]
@hamogu , given your collaboration with NumFOCUS on updated code of conduct. Is this still relevant?

**Comment by hamogu** [comment]
It might still be relevant. NumFOCUS is moving very slowly, but depending on where they land we might or might not go with what they work out; so I suggest to leave this open until we know.

**Comment by kelle** [comment]
This was my idea and I no longer think it's a good idea so I'm choosing to close. We can reopen if anyone feels strongly.
